data/announcements/scheduled.yaml

```
@@ -217,3 +217,16 @@ announcements:
     The legacy Linux package repositories (`apt.kubernetes.io` and `yum.kubernetes.io` AKA `packages.cloud.google.com`)<br/>
     have been frozen starting from September 13, 2023 **and are going away in January 2024**, users *must* migrate.<br/>
     Please read our [announcement](/blog/2023/08/31/legacy-package-repository-deprecation/) for more details.
+
+- name: Kubecon 2023 EU
```

Year needs updating on this line.

```suggestion
- name: Kubecon 2024 EU
```

---

Thanks for the review. I will add them and include a screenshort too as requrested.